hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/TestCSMMetrics.java

```diff
@@ -132,10 +129,10 @@ static void runContainerStateMachineMetrics(
       assertCounter("WriteChunkMsNumOps", 0L, metric);
       double applyTransactionLatency = getDoubleGauge(
           "ApplyTransactionNsAvgTime", metric);
-      assertTrue(applyTransactionLatency == 0.0);
+      assertEquals(0.0, applyTransactionLatency, 0.0);
```

Comment:
Wouldn't Math.ulp be better for epsilon here? What does 0.0 mean as the epsilon here?

Reply:
IDEA simplified the assertion to this one.  I think it's the same as `assertEquals(0.0, applyTransactionLatency)`, without any `delta`.